adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/data/StreamState.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.adapters.base.data;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Collectors;
+import javax.management.openmbean.CompositeData;
+
+/**
+ * Representation of the stream state data
+ */
+public class StreamState
+{
+    Collection<SessionInfo> sessions;
```

Comment:
1. `private final`?
2. It does not buy anything with using `Collection`. How about changing it to `List`?